package.json

```diff
@@ -12,13 +13,13 @@
   },
   "scripts": {
     "analyze": "webpack src/index.js dist/redux-form.js -p --bail --profile --json > stats.json && webpack-bundle-analyzer stats.json",
-    "build": "npm run build:lib && npm run build:es && npm run build:umd && npm run build:umd:min && npm run build:flow",
-    "build:lib": "babel src --out-dir lib --ignore __tests__",
+    "build": "npm run clean && npm run build:lib && npm run build:es && npm run build:umd && npm run build:umd:min && npm run build:flow",
+    "build:lib": "babel src --out-dir . --ignore __tests__",
     "build:es": "cross-env BABEL_ENV=es babel src --out-dir es --ignore __tests__",
-    "build:flow": "cp src/*.js.flow lib && cp src/selectors/*.js.flow lib/selectors && cp src/*.js.flow es && cp src/selectors/*.js.flow es/selectors",
+    "build:flow": "cp src/*.js.flow . && cp src/selectors/*.js.flow selectors && cp src/*.js.flow es && cp src/selectors/*.js.flow es/selectors",
     "build:umd": "cross-env NODE_ENV=development webpack src/index.js -o dist/redux-form.js",
     "build:umd:min": "cross-env NODE_ENV=production webpack src/index.js -o dist/redux-form.min.js",
-    "clean": "rimraf dist lib es",
+    "clean": "rimraf $(cd src; ls) dist lib es",
```

Comment:
Oh yeah.  I think I'll just write a JS script to do it

Reply:
Also the point of that part of the script removes all files in the root directory that had been transpiled over from `src`, rather than removing `src` itself.

Reply:
And I thought bash on Windows is a thing now?

Reply:
a lot of the other package scripts use `cp`, `&&`, etc... those won't work in a non-sh-based shell in windows will they?

Reply:
Okay, yeah I figured if other scripts aren't compatible it wasn't worth fixing this one right now.  No worries, I had never seen or used this `$(cd src; ls)` idiom before either, I was surprised that's all it took to accomplish what I needed to do.